include/pgduckdb/pgduckdb_background_worker.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "duckdb.hpp"
+#include "duckdb/parser/parsed_data/create_info.hpp"
+#include "duckdb/parser/parsed_data/create_info.hpp"
+#include "duckdb/common/unordered_set.hpp"
+#include "duckdb/parser/column_definition.hpp"
+#include "duckdb/parser/constraint.hpp"
+#include "duckdb/parser/statement/select_statement.hpp"
+#include "duckdb/catalog/catalog_entry/column_dependency_manager.hpp"
+#include "duckdb/parser/column_list.hpp"
+#include "duckdb/parser/parsed_data/create_table_info.hpp"
+
+extern "C" {
+#include "postgres.h"
+}
+
+void DuckdbInitBackgroundWorker(void);
+
+namespace pgduckdb {
+
+void SyncMotherDuckCatalogsWithPg(bool drop_with_cascade);
```

Comment:
Since it uses `__md_local_databases_metadata()` and the motherduck specific catolog version stuff, I think for now it makes sense to keep MotherDuck in the name. If/when we change it to support plain DuckDB I think it makes sense to rename it.